ebuild-writing/common-mistakes/text.xml

```diff
@@ -378,6 +378,17 @@ Make sure when you bump a version, the stable KEYWORDS are all marked as
 <c>~</c>.
 </p>
 
+</body>
+</subsection>
+<subsection>
+<title>Unused flags in IUSE</title>
```

Comment:
While at it, why not include eclasses ;) especially the infamous *eutils.eclass*!
```suggestion
<title>Unused USE flags and eclass inherits in the ebuild</title>
```

Reply:
Great idea!